docs/fx-core/bot-help.md

```diff
@@ -32,3 +32,24 @@ Suppose an existing AAD has been registered, you can reuse this AAD by configuri
     }
 ```
 4. Provision the Bot, then it will reuse this AAD and won't create a new one.
+
+## How can I trouble shoot issues when Teams Bot isn't responding on Azure?
+ 
+For general trouble shooting guidance, please refer to [Bot Framework Trouble Shooting Index](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-troubleshoot-index?view=azure-bot-service-4).
+
+When Teams Bot isn't responding, you probably want to check the its status on Azure. There're two fundamental points you'd better check first:
```

Comment:
check **the status** on Azure. There **are** two fundamental

Reply:
Good catch for the status - for the second, I believe `There're` is short for `There are`.